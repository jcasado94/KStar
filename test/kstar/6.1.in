k 1
k 2
k 5