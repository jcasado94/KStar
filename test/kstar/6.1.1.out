{
    "TestName": "6.1.1",
    "Paths": [
        {
            "Edges": [
                {
                    "U": 4,
                    "V": 5,
                    "I": 0
                },
                {
                    "U": 3,
                    "V": 4,
                    "I": 0
                },
                {
                    "U": 1,
                    "V": 3,
                    "I": 0
                },
                {
                    "U": 0,
                    "V": 1,
                    "I": 0
                }
            ],
            "Cost": 4
        }
    ]
}